tracer/src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -1240,5 +1256,102 @@ internal void CollectTelemetry(IConfigurationTelemetry destination)
                 exporterTelemetry.CopyTo(destination);
             }
         }
+
+        private static double? BuildSampleRate(in ConfigurationBuilder config)
```

Comment:
TIL we get sampling rates from other sources. How does this interact with a sampling rate from remote config? cc @zacharycmontoya

Reply:
Never mind, the remote override happens later in `ImmutableTracerSettings`.
https://github.com/DataDog/dd-trace-dotnet/blob/6bb6368d49eb3050c738e0bfcf9bec4a009dfe50/tracer/src/Datadog.Trace/Configuration/ImmutableTracerSettings.cs#L323